common/djangoapps/student/roles.py

```diff
@@ -245,6 +245,33 @@ class OrgRole(RoleBase):
     def __repr__(self):
         return f'<{self.__class__.__name__}>'
 
+    def update_org_course_role(self, user, orgs):
```

Comment:
@farhaanbukhsh, should this be named `update_org_course_role` even though the docstring for the role says `A named role in a particular org independent of course`?